e2e/utils/zetacore.go

```diff
@@ -204,6 +204,27 @@ type waitConfig struct {
 	matchFunction func(tx crosschaintypes.CrossChainTx) bool
 }
 
+// WaitCctxRevertedByInboundHash waits until cctx is reverted by inbound hash.
+func WaitCctxRevertedByInboundHash(
+	ctx context.Context,
+	t require.TestingT,
+	hash string,
+	c CCTXClient,
+) crosschaintypes.CrossChainTx {
+	// criteria for reverted cctx
+	searchForCCTXReverted := Matches(func(tx crosschaintypes.CrossChainTx) bool {
```

Comment:
Can we use `MatchStatus()` ?

Reply:
Good point. I'll wrap this into a `MatchReverted` to check reverted cctx strictly.

Reply:
https://github.com/zeta-chain/node/pull/3025/commits/7a2880bf21e47dfa10d0f3a9dd3756064237afbd